.github/workflows/trigger-e2e-tests.yml

```
@@ -28,19 +32,50 @@ jobs:
+    name: "🚀 Bootstrap Tests"
+    if: github.event_name == 'schedule' || github.ref == 'refs/heads/master' || github.base_ref == 'master'
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        version: ["9.1.1", "9.0.5", "8.19.2", "8.18.5", "8.17.10", "8.16.6", "8.15.5", "8.14.3", "8.13.4", "8.12.2", "8.11.4", "8.10.4", "8.9.2", "8.8.2", "8.7.1", "8.6.2", "8.5.3", "8.4.3", "8.3.3", "8.2.3", "8.1.3", "8.0.1", "7.17.29", "7.16.3", "7.15.2", "7.14.2", "7.13.4", "7.12.1", "7.11.2", "7.10.2", "7.9.3"]
+        env: [docker, eck-3.1.0]
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)